### DISCUSSION TITLE
Graphs options to enhance graphs display

### DISCUSSION THREAD

**Discussion by alexisrc1** [sqlpage/SQLPage#590] [Ideas]
I am building a dash with SQLPage and here is an example of the graph that displays hourly revenue vs last year
![image](https://github.com/user-attachments/assets/a67b33cb-aa40-4db1-b54c-216fbdcdbfa0)

I'd like to be able to format the yaxis as k€, and the number on top of the bars should be k€ as well as be displayed above the graph for better visibility.
Can an option "above" be implemented for displaying numbers above the bars, and an option "k€" as well so that the graphs displays the yaxis and the labels as "k€" 

thanks !!

**Comment by lovasoa** [comment]
Displaying the inline labels above the bars would indeed be a useful option. 

For the unit, I think you should just divide the values by 1000? That would make your graph cleaner.

**Reply by alexisrc1** [reply]
yes the best I achieved is this : 
![image](https://github.com/user-attachments/assets/8508099b-4c13-4dbb-a888-66a631d19f4a)

**Reply by lovasoa** [reply]
It may look better with one less digit
